File: API/Entities/Person.cs
using System.Collections.Generic;

namespace Skillustrator.Api.Entities
{
    public class Person : EntityBase
    {
        public Person()
        {

        }
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public ICollection<PersonSkill> Skills { get; set; }

        public int? OrganizationId { get; set; }

        public Organization Organization { get; set; }
    }
}

File: UI/src/app/models/person.ts
import { PersonSkill } from './person-skill';
import { Organization } from './organization';

export class Person {
  id: number;
  firstName: string;
  lastName: string;
  skills: PersonSkill[];
  organization: Organization;

  constructor(id: number, firstName: string, lastName: string, skills: PersonSkill[], organization: Organization) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.skills = skills;
    this.organization = organization;
  }
}
